docs/modules/general/pages/implementation-details/mfe-content-loading/webcomponents.adoc

```diff
@@ -0,0 +1,195 @@
+
+IMPORTANT: Webcomponent expose method enables the OneCX platform to use UI Apps developed with different frameworks and versions. It is the **recommended method for all Applications**.
+
+NOTE: Web Components technologies are used within OneCX platform, however OneCX offers its own functionalities that should be used to expose Microfrontend's content using the webcomponent method. Those functionalities are based on Web Components technologies, so there is no need to get familiar with them. Nevertheless, more resources about Web Components can be found https://developer.mozilla.org/en-US/docs/Web/API/Web_components[here]
+
+== Web Components preview
+Web Components are used in OneCX context for dynamic content creation. OneCX platform uses Web Components technologies to bound UI components to html elements. OneCX platform will use information from Application Store to determine what html elements have to be rendered to allow certain modules and Remote Components to bound to page content.
+
+With webcomponents method OneCX platform is only loading the code required for Microfrontend's content into the Shell UI memory and creating a "host" element for Remote Components or modules to attach into using Web Components. Microfrontend's exposed content is partially responsible for the content creation. Microfrontends have to adapt exposed modules and components so they are using Web Components Custom Elements.
+
+=== Custom Elements
+Web Components technology called Custom Elements is set of tools used to achive the bounding between components and html elements. As mentioned, OneCX platform will create html elements specified via configuration whenever a module or Remote Component has to be displayed on a page. There are 2 steps that OneCX platform takes when dealing with content loading with webcomponent expose method:
+
+. Load expose content via module-federation into memory
+. Create html element on a page to which expose content should be bounded
+
+In the xref:implementation-details/mfe-content-loading/index.adoc#_module_federation_loading_explained[Microfrontend content loading] it was mentioned that when exposing Microfrontend content, files specified per content should take care of all setup needed to display it. With webcomponents method, those files need to contain source code, which will register Custom Element in the Custom Elements Registry. Without getting into the details of Web Components, a file used for content loading needs to use Web Components to assign some component to the html element that OneCX platform will render. This way when OneCX platform creates this html element on a page, Custom Elements Registry will use the assigned component and display it on a page.
+
+=== Webcomponent method requirements
```

Comment:
Webcomponent **expose** method requirements (Note: before I start leaving more similar comments: From my understanding it should be **Webcomponent expose method** ?)